elasticsearch/dictionaries/synonyms/en-US.txt

```diff
@@ -0,0 +1,47 @@
+# english dialects
+favorite, favourite
+
+# media
```

Comment:
why are you splitting this group in multiple synonyms instead of putting everything in a single line?

Reply:
discussed in meeting: hyponyms and hypernyms